README.rst

```diff
@@ -14,18 +14,78 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.2.6 or newer.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
```

Comment:
> maybe I'm being too picky

Probably good to be picky now if it helps reduce the number of Community questions from confused people in the longer term.